### ISSUE TITLE
limits of powers of constants

### ISSUE THREAD

**Issue by rtoy** [rtoy/maxima#3571]
Imported from SourceForge on 2024-07-08 09:28:50
Created by **[willisbl](https://sourceforge.net/u/willisbl/)** on 2014-11-22 13:16:28
Original: https://sourceforge.net/p/maxima/bugs/2847

---

~~~~
Complex infinity might be better than und:

    (%i1) limit((1+%i)^n,n,inf);
    (%o1)                                 und

No need for an error:
    (%i2) limit((5+%i)^n,n,inf);

    sign: argument cannot be imaginary; found %i

    Maxima branch_5_34_base_129_g62902ed_dirty http://maxima.sourceforge.net
    using Lisp Clozure Common Lisp Version 1.9-r15764  (WindowsX8632)


~~~~

**Comment by rtoy** [comment]
Imported from SourceForge on 2024-07-08 09:28:52
Created by **[dgildea](https://sourceforge.net/u/dgildea/)** on 2014-12-10 21:27:11
Original: https://sourceforge.net/p/maxima/bugs/2847/#1e23

---

- **status**: open --> closed
- **assigned_to**: Dan Gildea

**Comment by rtoy** [comment]
Imported from SourceForge on 2024-07-08 09:28:55
Created by **[dgildea](https://sourceforge.net/u/dgildea/)** on 2014-12-10 21:27:11
Original: https://sourceforge.net/p/maxima/bugs/2847/#4fe7

---

Fixed by commit [f1d769c05ae3ea921b1f95461ef530cb6782102f]
